src/shared/ReactSharedInternals.js

```diff
@@ -10,15 +10,17 @@ import React from 'react';
 const ReactSharedInternals =
   React.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED;
 
+const hasOwnProperty = Object.prototype.hasOwnProperty;
```

Comment:
This is due to a newly recommended rule, https://eslint.org/docs/rules/no-prototype-builtins